pkgs/top-level/go-packages.nix

```diff
@@ -0,0 +1,55 @@
+/* This file defines the composition for Go packages. */
+
+{ pkgs, overrides, stdenv }:
+
+let self = _self // overrides; _self = with self; {
```

Comment:
This set must have also a `go` attribute, to know which go version are we using for this set of packages.